posts/redesigning-tracks-chicago-sloan.md

```diff
@@ -0,0 +1,37 @@
+
+**Develop comprehensive automated-analysis coverage of core exercises:** We want a student to be able to pass through the core of a track rapidly. In designing exercises, we will deliberately focus on how we can provide 95% automated analysis coverage to the possible solutions. This means that the interactions that mentors have will be with the 5% of students that really need their help, or on more interesting and varied side exercises. As part of this, we want to develop tools that enable our mentors to contribute to the automated analysers without having to dig into the codebase.
+
+**Offer bespoke automated mentoring and mentoring hints based on previous experience:** Once we have completed the previous steps, we want to explore tailoring the feedback that a student gets to their previous experience, either in terms of the automated responses they receive or in terms of hints we give to mentors. The feedback someone needs when learning a specific language depends heavily on their previous knowledge. Their anchor points are different, and their preexisting knowledge might be harder to unlearn, so by considering someone's previous experience, we can offer a personalised and more useful experience for students.
+
+**Adding in-browser coding support:** In parallel to the rethinking of core tracks, we are going to add one of our most-requested features: the ability to solve exercises in the browser. This will not replace using the CLI, which we believe is an incredibly valuable way to learn, but will offer people the ability to quickly try and explore languages without having to install that language and its tooling.
+
+### The Funding
+
+The funding we are receiving from The Sloan Foundation via James and Gary will allow Jeremy, Nicole, Charles and Karlo to invest most of their time for the next 12 months into the product work, development work, and architecture required to achieve the aims above.
+
+It will also allow us to hire Erik to lead on the "curriculum" side of the project. Erik has been a significant contributor to Exercism since 2016, maintaining the C#, F# and Scala tracks, building analysers, designing exercises, contributing to project-specifications and mentoring. He is also passionate about programming languages and has completed the majority of tracks on Exercism. He is also crucially someone who is universally liked and respected amongst the Exercism community, and therefore perfectly positioned to help navigate us through understanding how we can design tracks and systems that allow the differences and similarities of 50 different programming languages to shine through.
```

Comment:
```suggestion
It will also allow us to hire Erik to lead on the "curriculum" side of the project. Erik has been a significant contributor to Exercism since 2016, maintaining the C#, F# and Scala tracks, building analysers, designing exercises, contributing to project-specifications and mentoring. He is also passionate about programming languages and has completed a significant proportion of the tracks on Exercism. He is also crucially someone who is universally liked and respected amongst the Exercism community, and therefore perfectly positioned to help navigate us through understanding how we can design tracks and systems that allow the differences and similarities of 50 different programming languages shine through.
```